int8 IMG_INTRINSICS=0
int8 IMG_EXTRINSICS_RTOL=1
int8 IMU_INTRINSICS=2
int8 IMU_EXTRINSICS=3
int8 key
---
string value
